soc/nordic/nrf54h/cpurad_mgmt.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file Nordic Semiconductor nRF54h20 processors family management helper for the radio core.
+ */
+
+#include <zephyr/init.h>
+
+#include <ironside/se/cpuconf.h>
+
+static int nrf54h20_cpurad_init(void)
+{
+	uint32_t dummy = 0xbadcafe;
```

Comment:
Should we comment on why this is here? (missing boot report)

Reply:
fixed